src/sentry/static/sentry/app/views/performance/constants.tsx

```diff
@@ -8,6 +9,9 @@ export const PERFORMANCE_TERMS: Record<string, string> = {
   errorRate: t(
     'Error rate is the percentage of recorded transactions that had a known and unsuccessful status.'
   ),
+  p50: t('p50 indicates the duration that 50% of transactions were faster than.'),
```

Comment:
This reads a bit confusing, at least to me -- should `were` be present-tense? Can `than` at the end be dropped, otherwise, to me, it seems like it needs what it's faster than.

p50 indicates the duration at which 50% of transactions are faster
p50 indicates the duration that 50% of transactions are faster
p50 indicates the duration where 50% of transactions are faster

Reply:
I like 'p50 indicates the duration that 50% of transactions are faster'